### PR TITLE
add custom delimiter option

### PR DESCRIPTION
**Pull request by luckman212**
### This is a verbatim copy of my old PR from main (abandoned?) fork - see https://github.com/zmwangx/ets/pull/17
---
One problem that I had was the delimiter of a single space was making it hard to integrate with some other logging tools that I also use.  I wanted to use `\t` tab instead, but I went ahead and made this PR that adds a `-d / --delim` flag to allow any custom delimiter that might be desired.

Examples
```shell
ets --delim ' - '
ets -d '\t\t'
ets -d ':  '
```

One little neat trick you can do with this is create a delimiter that causes the terminal to beep when new lines are logged, e.g.

```shell
$ ping 1.1.1.1 | ets -d '\t\x07'
```

Hope this patch is ok, but let me know 👍

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -214,6 +216,16 @@ Options:
 			*format = "[%T]"
 		}
 	}
+	if *delim == "" {
+		*delim = "\t"
```
Comment (gdubicki) [inline main.go]: Hey @luckman212, thanks for the PR!

Unless there's a good reason for it, I would suggest to not change the default delimiter from space to tab though, for backward compatibility.

Apart from that, it looks good to me!

**Review by gdubicki — CHANGES_REQUESTED** [review]

File: main.go (hunk 1 above)
Comment (luckman212) [reply main.go]: Great, I just pushed another update and squashed the commits, so hopefully this one is good to go

**Comment by gdubicki** [comment]
Thanks again! I’ll make a new release tomorrow.

**Comment by gdubicki** [comment]
I need a bit more time for the release - updating the man page is ~a nightmare~ an „interesting” time-travel-like experience… 🦕😅

**Pull request merged**